.gitignore

```diff
@@ -9,6 +9,4 @@ coverage.txt
 
 noise
 chat
-debug
-
-go.sum
```

Comment:
Not a fan of this go.sum, it doesn't add any benefit and will be different for every developer. Should keep it out.

Reply:
https://github.com/golang/go/wiki/Modules#should-i-commit-my-gosum-file-as-well-as-my-gomod-file

Reply:
That link has been referenced in several PR's, but the claims seem incredibly weak to me. They basically don't trust github at google, so other projects have to suffer the consequences.